internal/statemachine/testdata/image_definitions/test_bad_class.yaml

```
@@ -13,7 +13,8 @@ gadget:
 model-assertion: pi-generic.model
 rootfs:
   seed:
-    url: "https://git.launchpad.net/~ubuntu-core-dev/ubuntu-seeds/+git/ubuntu"
+    urls:
+      - "https://git.launchpad.net/~ubuntu-core-dev/ubuntu-seeds/+git/ubuntu"
```

I know this is just test data used for testing bad yaml cases, but maybe we should adjust it to use the right urls after the redesign. This would result in an invalid seed url, since germinate would append the flavor (which by default is `ubuntu`) and it would fail anyway. The same is for many other urls in `testdata/`.

```suggestion
      - "https://git.launchpad.net/~ubuntu-core-dev/ubuntu-seeds/+git/"
```